resources/other/userAgents.json

```diff
@@ -7,13 +7,18 @@
 	"separator",
 	{
 		"identifier": "chrome",
-		"title": "Mask as Google Chrome 50 (Windows)",
-		"value": "Mozilla/5.0 (Windows NT 10.0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/50.21.2661.78 Safari/537.36"
+		"title": "Mask as Google Chrome 55 (Windows)",
+		"value": "Mozilla/5.0 (Windows NT 10.0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/55.0.2869.0 Safari/537.36"
+	},
+	{
+		"identifier": "mobile",
+		"title": "Mask as Google Chrome 60 (mobile device)",
+		"value": "Mozilla/5.0 (Linux; Android 7.1.1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/55.0.2869.0 Mobile Safari/537.36"
```

Comment:
*mobile* is too generic to be used as identifier of specific UA, I would call it *chromeMobile* and use *Android* instead of *mobile device* in description.